cmake/os_detection.cmake

```diff
@@ -26,6 +26,7 @@ macro(DEFINE_OS)
       OUTPUT_VARIABLE UNAME_OUT)
   string(TOLOWER "${UNAME_OUT}" OS_VERSION)
 
+  set(OS_VERSION "preempt-rt")
```

Comment:
Seems that you forgot to modify this file, see below as well.

Reply:
+1